.github/workflows/release-tarball.yaml

```diff
@@ -5,6 +5,7 @@ concurrency:
   cancel-in-progress: true
 
 on:
+  pull_request:
```

Comment:
I thought that we would not release on pull_request, or?

Reply:
yes this was just for testing reason